### PR TITLE
Address id review

### LINKED ISSUE #42: ID review
I've opened #43 to make some basic ID edits. The comments below are for conceptual questions and other issues I wasn't sure how best to address.

The title of the guide is not showing up on the draft site for some reason. 

## What you'll learn
- In the PR, I reordered some of the info so that the discussion of Operators in general comes before references to the OL and Strimzi Operators. I think there could still be a little more introductory info about operators, even just a sentence or two more, similar to the info for OpenShift. I noticed there was also a short paragraph about Kubernetes in the https://openliberty.io/guides/cloud-openshift.html[Deploying microservices to OpenShift] guide, not sure if that info would be useful here as well? 

- `The inventory microservice subscribes to that information so that it can keep an updated list of all the systems and their current system loads.`
Is there more than one system? This is confusing because the microservice is "system" singular, but the `inventory` microservice lists more than one "system"? Or are there two different uses of the term system here? If so, should distinguish one from the other somehow

- `The current inventory of systems can be accessed via the /systems REST endpoint. `
Is this part of the guide? If so better to discuss when you get to it. If not do you need to mention it? Seems unconnected to the other information here. It might be better to move it to the "Accessing..." section, where the user actually accesses the REST endpoint.

- There are a number of steps in this guide- installing, deploying, accessing, scaling. It might be helpful to give the user a brief overview of the sub-tasks so they can orient themselves as they work through the guide.

## Additional prerequisites

- `Before you can deploy your microservices, you must gain access to a cluster on OpenShift and have an OpenShift client installed.`
There is discussion about how to get access to OS cluster, but installing the OS client isn't explained anywhere, why or how.

- `You should see an asterisk (*) beside the project guide.`
Sample results would be helpful here

- the https://openliberty.io/guides/cloud-openshift.html[Deploying microservices to OpenShift] guide has a prereq for docker as well- is that needed here? It also has a prereq for the CLI, which is needed here.

## Installing the Operators
- `To install the two Operators, navigate to the web console, then to Operators > OperatorHub on the sidebar. `
This is the first mention of "the web console" in this guide. There's no explanation of what it is or how to find it. I know this means the OS console but the guide should introduce it somehow. Is this the only step that requires the console? Can the installation be done from the CLI? The user should know why they're going to the console only for this step (if that's the best way to install).

- `The Open Liberty Operator introduces new CRDs, including the OpenLibertyApplication.`
Looking at the example results, `OpenLibertyApplication` is the "KIND", while the name is `openlibertyapplications`. There is inconsistency throughout the guide between `OpenLibertyApplication` in some places and `OpenLibertyApplications` in others, but from looking at these results the name of the CRD is `openlibertyapplications`. Confirm which name is correct and use it consistently

- `Run the following command to view all the supported API resources available through the Open Liberty Operator:`
...
`You see the following output, which shows the custom resource definitions (CRDs) that are installed by the Open Liberty Operator`

In the command, you  view the "resources" that are "available", but the output shows the "CRDs" that are "installed". Are all the available resources also installed? It's not clear. Also, this section is about installing the Operators. If the Operators install resources there needs to be some explanation of that. The distinction between Operators and resources isn't clear. Same issue with the Strimzi Operator.

## Deploying the system and inventory applications to OpenShift

- `To deploy the Open Liberty system and inventory applications...`
In the introduction, `system` and `inventory` are referred to as "microservices". Here they are "applications". Pick one term and use it  consistently.

- `One of the benefits of using ImageStream is that the operator redeploys the application when it detects a new image is pushed into the ImageStream`
What is `ImageStream`. Is it the same as the image stream object mentioned earlier? It's not defined here or in the example yaml. If it is image stream object, use that term consistently.

- `Update the value of the bootstrap address variable to the required values. You can find the bootstrap address by running the following command:`
Example results would be helpful here. 
- `You can also substitute OpenLibertyApplications with the short form olapps.` If its useful to use shortnames you should mention/explain them back where they are listed in CRDs. 

## Scaling up the system application

- `Revisit the inventory application and you now see three elements`
Should be more clear that the user views the rest endpoint here through the same URL in the previous step. Also, is there a more precise term to use than "elements"? Perhaps "replicas"? "instances?

### PR DESCRIPTION
**Pull request by AustinSeto**
Closes #42

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -145,21 +156,23 @@ openlibertydumps          oldump,oldumps     openliberty.io   true         OpenL
...
-Run the following command to view all the supported API resources that are available through the Strimzi Operator:
+Run the following command to view all CRDs used by the Strimzi Operator:
 
 [role='command']
 ```
 oc api-resources --api-group=kafka.strimzi.io
 ```
 
-Look for the following output, which shows the CRDs that are installed by the Strimzi Operator:
+Look for the following output, which shows the CRDs that are used by the Strimzi Operator:
```
Comment (dmuelle) [inline README.adoc:175]: suggested edit

Look for the following output, which lists the CRDs that the Strimzi Operator uses, along with object kinds and shortnames.

File: README.adoc [hunk 2]
```diff
@@ -271,7 +284,7 @@ Create a build template to configure how to build your images.
 ----
 
 The [hotspot file=0]`build.yaml` template includes two objects. 
-The [hotspot=imageStream file=0]`image stream` object provides an abstraction from the image in the image registry. 
+The [hotspot=imageStream file=0]`ImageStream` object `kind` provides an abstraction from the image in the image registry.
```
Comment (dmuelle) [inline README.adoc:287]: no need to monospace "kind"

File: README.adoc [hunk 3]
```diff
@@ -283,15 +296,15 @@ a [hotspot=docker file=0]`Docker` strategy build, which invokes the `docker buil
...
+Run the following commands to create the objects for the `system` and `inventory` microservices:
 
 [role='command']
 ```
 oc process -f build.yaml -p APP_NAME=system | oc create -f -
 oc process -f build.yaml -p APP_NAME=inventory | oc create -f -
 ```
 
-Next, run the following commands to view the newly created image streams and build configurations for each application:
+Next, run the following commands to view the newly created `ImageStream` objects and build configurations for each microservice:
```
Comment (dmuelle) [inline README.adoc:307]: suggested edit:

Next, run the following commands to view the newly created `ImageStream` objects and the build configurations for each microservice:

File: README.adoc [hunk 4]
```diff
@@ -404,9 +417,9 @@ the Open Liberty Operator.
 The `applicationImage` parameter defines what container image is deployed as part of the `OpenLibertyApplication` CRD. 
 This parameter follows the `<project-name>/<image-stream-name>[:tag]` format.
 The parameter can also point to an image hosted on an external registry, such as Docker Hub. 
-The [hotspot=system file=1]`system` application is configured to use the [hotspot=sysImage file=1]`image` created from the earlier build. 
+The [hotspot=system file=1]`system` microservice is configured to use the [hotspot=sysImage file=1]`image` created from the earlier build. 
 
-One of the benefits of using `ImageStream` is that the operator redeploys the application when it detects a new image is pushed into the `ImageStream`.
+One of the benefits of using `ImageStream` objects is that the operator redeploys the application when it detects a new image is pushed into the `ImageStream`.
```
Comment (dmuelle) [inline README.adoc:422]: is it correct to say:

One of the benefits of using `ImageStream` objects is that the operator redeploys the application when it detects a new image is pushed into the `ImageStream` objects

Seems strange but should be consistent if there is one meaning for  `ImageStream`

File: README.adoc [hunk 5]
```diff
@@ -418,14 +431,21 @@ bootstrap address by running the following command:
...
+```
+
+The [hotspot=inventory file=1]`inventory` microservice is configured similarly to the [hotspot=system file=1]`system`
+microservice. Additionally, the `inventory` microservice includes the [hotspot=service file=1]`service` and
 [hotspot=expose file=1]`expose` parameters. The [hotspot=servicePort file=1]`service.port` parameter specifies which port
-is exposed by the container, allowing the application to be accessed from outside the container. To access the application
+is exposed by the container, allowing the microservice to be accessed from outside the container. To access the microservice
 from outside of the cluster, it must be exposed by setting the [hotspot=expose file=1]`expose` parameter to `true`.
-Exposing the application lets the Operator automatically create and configure routes for external access to your application.
+Exposing the microservice lets the Operator automatically create and configure routes for external access to your microservice.
```
Comment (dmuelle) [inline README.adoc:446]: suggested edit:

After you expose the microservice, the Operator automatically creates and configures routes for external access to your microservice.

File: README.adoc [hunk 6]
```diff
@@ -439,7 +459,7 @@ Next, run the following command to view your newly created `OpenLibertyApplicati
 oc get OpenLibertyApplications
 ```
 
-You can also replace `OpenLibertyApplications` with the shortname `olapps`.
+You can also replace `OpenLibertyApplications` with its shortname `olapps`.
```
Comment (dmuelle) [inline README.adoc:462]: avoid showing possession by inanimate objects- 

You can also replace `OpenLibertyApplications` with the shortname `olapps`.

File: README.adoc [hunk 7]
```diff
@@ -562,7 +582,7 @@ Wait a few minutes, then run the following command to see the newly scaled up `s
 oc get pods
 ```
 
-Revisit the `inventory` application and you now see three elements:
+Revisit the `inventory` microservice and you will now see three instances of the `system` microservice listed in the `inventory` endpoint:
```
Comment (dmuelle) [inline README.adoc:585]: avoid future tense:

Revisit the `inventory` microservice and you can now see three instances of the `system` microservice listed in the `inventory` endpoint:

**Review by dmuelle — CHANGES_REQUESTED** [review]
Looks good- I've made some suggestions per general ID guidance against using either future or subjunctive tense. Hard to avoid it in the "What you'll learn" section but outside of that should rephrase to avoid saying what "will" or "should" happen, per my suggestions.

Had a couple other comments for clarity & consistency- feel free to ping me w/ any questions or to discuss, thanks!

File: README.adoc (hunk 4 above)
Comment (AustinSeto) [reply README.adoc:422]: Perhaps saying just, "One of the benefits of using `ImageStream` objects is that the operator redeploys the application when it detects a new image is pushed" or is this too vague?

**Comment by AustinSeto** [comment]
Thank you for the review @dmuelle, I have addressed all the changes and pushed them to the LGDEV site.

**Review by dmuelle — APPROVED** [review]
LGTM- Thanks for making the changes

**Pull request merged**